src/pubsub/ua_pubsub_writergroup.c

```diff
@@ -1096,6 +1112,20 @@ UA_WriterGroup_publishCallback(UA_Server *server, UA_WriterGroup *writerGroup) {
         UA_PublishedDataSet *pds =
             UA_PublishedDataSet_findPDSbyId(server, dsw->connectedDataSet);
         if(!pds) {
+            if (UA_NodeId_isNull(&dsw->connectedDataSet)) {
```

Comment:
Check for code formatting adhere to the rules of the stack.

e.g. `if` statements do not have a following space.

Reply:
You're right. Fixed by ec25477.